sample/WorkerServiceDemo/Worker.cs

```diff
@@ -19,6 +19,14 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
         {
             _logger.LogInformation("Worker started");
 
+            // Logging child property Name of structured object structured
+            // to a separate column according to configuration in AddictionalColumns in appsettings.json
```

Comment:
AdditionalColumns

Reply:
:D those columns are so addictional, i can't get rid of them. Fixed.